src/main/java/com/google/cloud/tools/managedcloudsdk/MessageListenerForwardingHandler.java

```diff
@@ -33,7 +33,7 @@ public void bytes(byte[] bytes, int length) {
   }
 
   @Override
-  public Void getResult() {
+  public String getResult() {
     return null;
```

Comment:
This looks like it should be an abstract method.

Reply:
It's an artifact of generalizing, but I moved this out and made it package private with the "command" package.